vars/buildPlugin.groovy

```diff
@@ -103,22 +99,10 @@ def call(Map params = [:]) {
                                 if (javaLevel) {
                                     mavenOptions += "-Djava.level=${javaLevel}"
                                 }
-                                if (params?.findbugs?.run || params?.findbugs?.archive) {
-                                    mavenOptions += "-Dfindbugs.failOnError=false"
```

Comment:
`failOnError` is not longer added to the build. With the current job implementation the build will not really reach the publishing stage with warnings NG if there is a FindBugs issue

Reply:
confirmed in https://ci.jenkins.io/job/Plugins/job/slack-plugin/job/PR-707/1/

Reply:
Fixed that but now the build is green, it should be unstable instead:

https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fslack-plugin/detail/PR-707/4/pipeline

Reply:
I think the maven configuration of FindBugs/SpotBugs should not be done in the Jenkinsfile. This is something that should be part of the Plugin POM.

Reply:
it's configured in the pom already:

https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L68-L70